platform/ui/src/components/Icon/getIcon.jsx

```diff
@@ -1,37 +1,42 @@
 import React from 'react';
 // Icons
+
 import arrowDown from './../../assets/icons/arrow-down.svg';
-import seriesActive from './../../assets/icons/series-active.svg';
-import seriesInactive from './../../assets/icons/series-inactive.svg';
 import calendar from './../../assets/icons/calendar.svg';
 import cancel from './../../assets/icons/cancel.svg';
 import chevronDown from './../../assets/icons/chevron-down.svg';
 import chevronRight from './../../assets/icons/chevron-right.svg';
+import iconNavStudies from './../../assets/icons/icon-nav-studies.svg';
 import infoLink from './../../assets/icons/info-link.svg';
 import launchArrow from './../../assets/icons/launch-arrow.svg';
 import launchInfo from './../../assets/icons/launch-info.svg';
+import listBullets from './../../assets/icons/list-bullets.svg';
 import logoOhifSmall from './../../assets/icons/logo-ohif-small.svg';
 import magnifier from './../../assets/icons/magnifier.svg';
 import notificationwarningDiamond from './../../assets/icons/notificationwarning-diamond.svg';
+import panelLeft from './../../assets/icons/panel-left.svg';
+import panelRight from './../../assets/icons/panel-right.svg';
```

Comment:
These icons could be used for something other than panel. Maybe `pushLeft` and `pushRight`

Reply:
I liked your idea. just updated .